webapp/src/main/resources/properties/en.properties

```diff
@@ -701,4 +701,37 @@ textUnit.gitBlameModal.close=Close
 textUnit.gitBlameModal.screenshots=Screenshots
 
 # Text for link to open screenshot modal in git-blame modal
-textUnit.gitBlameModal.screenshotModalOpen=View
\ No newline at end of file
+textUnit.gitBlameModal.screenshotModalOpen=View
+
+# Tooltip for button to open the translation history modal
+workbench.translationHistoryModal.info=Text unit history
+
+# Title of the translation history modal dialog
+workbench.translationHistoryModal.title=Text Unit Translation History
+
+# Label for the button to close the translation history modal dialog
+textUnit.translationHistoryModal.close=Close
+
+# Text in the section in the translation history dialog where info about the translation unit is given
+textUnit.translationHistoryModal.locale=Translation history for locale: {locale}
+
+# Text in the section in the translation history dialog where info about the translation unit is given
+textUnit.translationHistoryModal.source=Source string: "{source}"
```

Comment:
I'd have just "Source" and remove "string"

Reply:
Done